km/km_main.c

```diff
@@ -481,6 +495,12 @@ km_parse_args(int argc, char* argv[], int* argc_p, char** argv_p[], int* envc_p,
             km_set_snapshot_output_path(optarg);
             break;
          case 'm':
+            if (km_mgtdir != NULL) {
+               km_errx(1,
```

Comment:
> Why mutually exclusive instead of allowing command line to override environment variable?

Originally I had it so that the environment variable had priority over the command line flags and a warning was issued that the command line flag was being ignored.

Serge's thinking is: since we are probably running this in a container and since getting debug info out of a container is sometimes difficult it is better to just to fail immediately rather then letting the run continue with snapshot related values that are unexpected.